docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,145 @@
 ZWO Camera
 ==========
 
+This service operates a ZWO camera. There are currently four different types of ZWO cameras being used on the testbed:
+- ZWO ASI533MM (Science camera, https://www.zwoastro.com/product/asi533mm-mc/)
+- ZWO ASI290MM (TA camera, https://agenaastro.com/zwo-asi290mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI178MM (MOWFS Camera and Pupil Camera, https://agenaastro.com/zwo-asi178mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI1600MM (Phase Retrieval Camera, https://agenaastro.com/zwo-asi1600mm-p-cmos-monochrome-astronomy-imaging-camera-pro.html)
+
+For camera specs, see the website links above.
+
 Configuration
 -------------
 
+.. code-block:: YAML
+
+    science_camera:
+        service_type: zwo_camera
+        simulated_service_type: camera_sim
+        interface: hicat_camera
```

Comment:
Here too, replace the hicat reference with either a default proxy that lives inside of catkit2, or if there is none, just delete this line (a dedicated proxy is not required for a service).